imports/ui/accounts/Account.jsx

```diff
@@ -61,15 +65,18 @@ export default class AccountDetails extends Component{
             }
 
             if (result){
-                // console.log(result);
+                let totalAmount = [];
+                let amountAvailable = [];
+
                 if (result.available){
-                    const amount = result.available.amount || '0';
-                    const denomType  = result.available.denom;
 
+                    amountAvailable = cloneDeep(result.available);
+                    totalAmount = cloneDeep(result.available);
```

Comment:
What is the `totalAmount` is for?

Reply:
I have applied suggested changes in `a9da9c0 ` commit